Assets/GothicVR/Scripts/Player/Menu/MenuButtonScript.cs

```diff
@@ -0,0 +1,19 @@
+using GVR.Manager;
+using UnityEngine;
+
+namespace GVR.Player.Menu
+{
+    public class MenuButtonScript : MonoBehaviour
+    {
+        public void PlayFunction()
+        {
+            GvrSceneManager.I.LoadWorld(ConstantsManager.I.selectedWorld, ConstantsManager.I.selectedWaypoint);
+        }
+
+        public void QuitGameFunction()
+        {
+            Application.Quit();
```

Comment:
There's actually a Daedalus function which is called when quitting the game. Don't know if really needed, but I know you... You'll love this information and will seek... right now... 😉😋